x-pack/legacy/plugins/spaces/public/management/_index.scss

```diff
@@ -0,0 +1,3 @@
+@import './components/confirm_delete_modal/confirm_delete_modal';
+@import './edit_space/enabled_features/index';
+
```

Comment:
Great catch -- it turns out that the Spaces `SectionPanel` does not rely on the styles in that stylesheet, because Spaces does not make use of the optional `iconType` prop, whereas the Security panel does. I'll see if we need it for security since you mentioned we're not importing it there either. 
I'll update to import this `.scss` file, as that's the least invasive change to this PR.

We should consider removing the optional `iconType` prop from the spaces version since it's not used at all, but I'll leave that for another PR.